indi-gpio/indi_gpio.cpp

```diff
@@ -187,21 +195,36 @@ bool INDIGPIO::Connect()
     }
 
     // Get all lines
+    #ifdef HAVE_LIBGPIOD_V2
     int lines = m_GPIO->get_info().num_lines();
+    #else
+    auto lines = m_GPIO->get_all_lines();
+    #endif
 
     m_InputOffsets.clear();
     m_OutputOffsets.clear();
 
     // Iterate through all lines
+    #ifndef HAVE_LIBGPIOD_V2
```

Comment:
This should be version 2, so #ifdef HAVE_LIBGPIOD_V2

Reply:
uuups, thanks. Nice catch